addons/web/static/src/views/basic_relational_model.js

```diff
@@ -298,10 +298,11 @@ export class Record extends DataPoint {
     /**
      * FIXME: memoize this at some point?
      * @param {string} fieldName
+     * @param {Object} modifiers
      * @returns {boolean}
      */
-    isReadonly(fieldName) {
-        const { readonly } = this.activeFields[fieldName].modifiers || {};
+    isReadonly(fieldName, modifiers) {
+        const { readonly } = modifiers || this.activeFields[fieldName].modifiers || {};
```

Comment:
I would not encourage this as it doesn't go in the same direction as the relational model v2 (where we want the model not to take care of node details)